som/display_test.go

```diff
@@ -0,0 +1,30 @@
+package som
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/gonum/matrix/mat64"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUMatrixSVG(t *testing.T) {
+	assert := assert.New(t)
+
+	const svg = `<h1>Done</h1><svg width="60" height="60"><polygon points="20.000000,18.660254 20.000000,1.339746 0.000000,1.339746 0.000000,18.660254 20.000000,18.660254 " style="fill:rgb(125,125,125);stroke:black;stroke-width:1"></polygon><polygon points="30.000000,35.980762 30.000000,18.660254 10.000000,18.660254 10.000000,35.980762 30.000000,35.980762 " style="fill:rgb(91,91,91);stroke:black;stroke-width:1"></polygon><polygon points="40.000000,18.660254 40.000000,1.339746 20.000000,1.339746 20.000000,18.660254 40.000000,18.660254 " style="fill:rgb(91,91,91);stroke:black;stroke-width:1"></polygon><polygon points="50.000000,35.980762 50.000000,18.660254 30.000000,18.660254 30.000000,35.980762 50.000000,35.980762 " style="fill:rgb(125,125,125);stroke:black;stroke-width:1"></polygon></svg>`
```

Comment:
This doesn't have to be a `const`. There is no harm in it being a `const`, but since we are using this var in one test it won't bring much improvements in terms of compile optimization, unless you are worried that `assert.Equal` might accidentally modify it :-)